src/main/java/com/zufar/icedlatte/review/converter/ProductReviewDtoConverter.java

```diff
@@ -5,41 +5,53 @@
 import com.zufar.icedlatte.openapi.dto.RatingMap;
 import com.zufar.icedlatte.review.dto.ProductRatingCount;
 import com.zufar.icedlatte.review.entity.ProductReview;
+import com.zufar.icedlatte.user.entity.UserEntity;
+import org.mapstruct.*;
 import org.springframework.data.domain.Page;
-import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.Optional;
 
-@Service
-public class ProductReviewDtoConverter {
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING,
+        uses = ProductReviewDtoConverter.class, unmappedTargetPolicy = ReportingPolicy.IGNORE, injectionStrategy = InjectionStrategy.FIELD)
+public interface ProductReviewDtoConverter {
 
-    public static final ProductReviewDto EMPTY_PRODUCT_REVIEW_RESPONSE =
-            new ProductReviewDto(null, null,null, null, null, null, null, null, null);
+    ProductReviewDto EMPTY_PRODUCT_REVIEW_RESPONSE =
+            new ProductReviewDto(null, null, null, null, null, null, null, null, null);
 
-    public ProductReviewDto toProductReviewDto(ProductReview productReview) {
-        return new ProductReviewDto(
-                productReview.getId(),
-                productReview.getProductId(),
-                productReview.getProductRating(),
-                productReview.getText(),
-                productReview.getCreatedAt(),
-                productReview.getUser().getFirstName(),
-                productReview.getUser().getLastName(),
-                productReview.getLikesCount(),
-                productReview.getDislikesCount());
+    @Mapping(target = "productReviewId", source = "id")
+    @Mapping(target = "productId", source = "productId")
+    @Mapping(target = "productRating", source = "productRating")
+    @Mapping(target = "text", source = "text")
+    @Mapping(target = "createdAt", source = "createdAt")
```

Comment:
Apparently, these can be removed as well

Reply:
fixed